packages/@holaplexui-react/src/components/Modal.tsx

```diff
@@ -59,22 +59,21 @@ export function Modal(props: ModalProps) {
             >
               <Dialog.Panel
                 className={clsx(
-                  'z-40 w-full max-w-md transform !overflow-visible rounded-2xl bg-gray-900  text-left align-middle transition-all',
-                  'scrollbar-thumb-rounded-full relative flex h-full max-h-screen w-full flex-col overflow-y-auto rounded-xl bg-gray-900  text-white shadow-md scrollbar-thin scrollbar-track-gray-900  sm:h-auto  sm:max-w-lg',
+                  'relative z-40 flex flex-col bg-white h-full transform !overflow-visible align-middle transition-all scrollbar-thumb-rounded-full overflow-y-auto rounded-md shadow-md scrollbar-thin max-h-screen max-w-md sm:h-auto sm:max-w-lg',
+                  'modal-content',
                   props.short ? 'sm:max-h-[30rem]' : 'sm:max-h-[50rem]',
                   props.scroll ? 'pt-6' : 'p-6'
                 )}
               >
                 <button
                   type="button"
                   onClick={() => props.setOpen(false)}
-                  className="absolute -top-2 -right-2 z-50 rounded-full bg-white p-1 hover:bg-gray-100 hover:text-gray-400"
+                  className="absolute top-1 right-1 z-50 rounded-full bg-gray-50 p-2 hover:bg-gray-100 hover:text-gray-400"
```

Comment:
styling of the button should be done through a class so this 1 `modal-close` set the shape color and hover states through the class